CHAP/__init__.py

```diff
@@ -0,0 +1,3 @@
+from CHAP.reader import *
```

Comment:
Keara, I suggest to avoid implicit imports in any places of python code since they may lead to unexpected behavior in development and integration with other packages. The import like this:
```
from CHAP.reader import *
```
means that all classes, functions global variables will be loaded from the `CHAP..reader` module. Now, imaging if somebody in other part of the code will have a class, function or variable which is identical to what is defined in `CHAP.reader`. This will overwrite previous import and code may not to do what you expect it from it. It is much safer to use explicit imports, e.g.:
```
from CHAP.reader import Reader, XXXReader, ...
```
Doing this way you know which classes, functions, and variables are loaded and will be used.

I strongly recommend to fix all implicit imports in favor of explicit ones.

Reply:
Okay, thanks! I will do this before merging (but probably not this afternoon).